electrochemistry.ttl

```diff
@@ -6463,616 +7060,332 @@ emmo:EMMO_a2a68924_b759_4ba4_a25d_126021569fb9 rdf:type owl:Class ;
-                                                                                           emmo:EMMO_c84c6752_6d64_48cc_9500_e54a3c34898d "https://en.wikipedia.org/wiki/Constant_current" ;
-                                                                                           skos:altLabel "ConstantCurrentSignal"@en ,
-                                                                                                         "DC"@en ;
-                                                                                           skos:prefLabel "DirectCurrent"@en .
-
-
-###  https://emmo.info/electrochemistry#electrochemistry_46676855_68b0_4096_ac6c_35400111d46d
-<https://emmo.info/electrochemistry#electrochemistry_46676855_68b0_4096_ac6c_35400111d46d> rdf:type owl:Class ;
-                                                                                           rdfs:subClassOf <https://emmo.info/electrochemistry#electrochemistry_ddade648_2033_47b6_bc36_b562a9af591e> ;
-                                                                                           emmo:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "plot of the time-dependent amount of electric charge passed through an electrochemical system due to the application of a electric potential"@en ;
-                                                                                           skos:altLabel "ChargeTimeCurve"@en ,
-                                                                                                         "ElectricChargeTimeCurve"@en ,
-                                                                                                         "ElectricChargeTimePlot"@en ,
-                                                                                                         "QtCurve"@en ;
-                                                                                           skos:prefLabel "ChargeTimePlot"@en .
-
-
-###  https://emmo.info/electrochemistry#electrochemistry_58a20764_c339_4856_983a_05092b5282e8
-<https://emmo.info/electrochemistry#electrochemistry_58a20764_c339_4856_983a_05092b5282e8> rdf:type owl:Class ;
-                                                                                           rdfs:subClassOf <https://emmo.info/electrochemistry#electrochemistry_0a03ce7e_d79f_412e_a103_b5d74de9f4d7> ,
-                                                                                                           [ rdf:type owl:Restriction ;
-                                                                                                             owl:onProperty emmo:EMMO_3446e167_c576_49d6_846c_215bb8878a55 ;
-                                                                                                             owl:someValuesFrom emmo:EMMO_4f2d3939_91b1_4001_b8ab_7d19074bf845
```

Comment:
Plots can also be linked to quantities. We can think about the right way to do it once the issues with the reasoner are resolved. I tried using `hasVariable`, but perhaps we can borrow from terminology using in plotting software, e.g. `AxisQuantity`, or something similar.